diff --git a/src/vulkan/device.rs b/src/vulkan/device.rs
index 021825e..1b02814 100644
--- a/src/vulkan/device.rs
+++ b/src/vulkan/device.rs
@@ -682,7 +682,7 @@ impl super::Device {
                         temp_options.debug_info = Some(naga::back::spv::DebugInfo {
                             source_code: &debug.source_code,
                             file_name: debug.file_name.as_ref(),
-                    language: naga::back::spv::SourceLanguage::WGSL,
+                    language: naga::back::spv::SourceLanguage::GLSL,
                         })
                     }
                     if !stage.zero_initialize_workgroup_memory {
@@ -1749,7 +1749,7 @@ impl crate::Device for super::Device {
                         .map(|d| naga::back::spv::DebugInfo {
                             source_code: d.source_code.as_ref(),
                             file_name: d.file_name.as_ref(),
-                    language: naga::back::spv::SourceLanguage::WGSL,
+                    language: naga::back::spv::SourceLanguage::GLSL,
                         });
                 if !desc.runtime_checks.bounds_checks {
                     naga_options.bounds_check_policies = naga::proc::BoundsCheckPolicies {
